### PR TITLE
Improve QuickPlots and Quickplots simple

### PR DESCRIPTION
**Pull request by awirb**
* Add nameOfAvalanche within plots (title or textbox) and file names
* Combine quickPlot and quickPlotSimple
* introduce nameofAvalanche in makeSimDict

### CODE REVIEW

File: avaframe/out3Plot/outQuickPlot.py [hunk 1]
```diff
@@ -27,22 +25,116 @@
...
+    ax1 = fig.add_subplot(131)
+    cmap = cmapGB
+    im1 = plt.imshow(data1, cmap=cmap, extent=[0, Lx, 0, Ly], origin='lower', aspect=nx/ny)
+    fig.colorbar(im1, ax=ax1)
+    ax1.set_aspect('auto')
+    title = str('%s' % name1)
+    ax1.set_title(title)
+    ax1.set_xlabel('x [m]')
+    ax1.set_ylabel('y [m]')
+    ax2 = fig.add_subplot(132)
```
Comment (fso42) [inline avaframe/out3Plot/outQuickPlot.py:77]:
```suggestion
    ax1.set_ylabel('y [m]')
    
    ax2 = fig.add_subplot(132)
```

File: avaframe/out3Plot/outQuickPlot.py [hunk 2]
```diff
@@ -27,22 +25,116 @@
...
+    im2 = plt.imshow(data2, cmap=cmap, extent=[0, Lx, 0, Ly], origin='lower', aspect=nx/ny)
+    fig.colorbar(im2, ax=ax2)
+    ax2.set_aspect('auto')
+    ax2.set_xlabel('x [m]')
+    title = str('%s' % name2)
+    ax2.set_title(title)
+    ax3 = fig.add_subplot(133)
```
Comment (fso42) [inline avaframe/out3Plot/outQuickPlot.py:84]:
```suggestion
    ax2.set_title(title)
    
    ax3 = fig.add_subplot(133)
```

File: avaframe/out3Plot/outQuickPlot.py [hunk 3]
```diff
@@ -27,22 +25,116 @@
...
+    im3 = plt.imshow(dataDiff, cmap=cmap,
+                     extent=[0, Lx, 0, Ly], origin='lower', aspect=nx/ny)
+    fig.colorbar(im3, ax=ax3)
+    ax3.text(nybox, nxbox, 'Mean: %.2f %s\n Max: %.2f %s\n Min: %.2f %s' %
+            (diffMean, unit, diffMax, unit, diffMin, unit), bbox=dict(boxstyle="square", ec='white', fc='white'),
+            horizontalalignment='left', verticalalignment='bottom')
+    ax3.set_aspect('auto')
+    ax3.set_xlabel('x [m]')
+    ax3.set_title('Difference ref-sim')
+    fig.savefig(os.path.join(outDir, 'Diff_%s_%s.%s' % (avaName, simName, outputFormat)))#, bbox_extra_artists=(suptitle,), bbox_inches="tight")
```
Comment (fso42) [inline avaframe/out3Plot/outQuickPlot.py:95]: I think the bbox_extra is needed, right?

File: avaframe/out3Plot/outQuickPlot.py (hunk 1 above)
Comment (awirb) [reply avaframe/out3Plot/outQuickPlot.py:77]: Done

File: avaframe/out3Plot/outQuickPlot.py (hunk 3 above)
Comment (awirb) [reply avaframe/out3Plot/outQuickPlot.py:95]: Done

File: avaframe/out3Plot/outQuickPlot.py (hunk 2 above)
Comment (awirb) [reply avaframe/out3Plot/outQuickPlot.py:84]: Done

**Review by fso42 — APPROVED** [review]

**Pull request merged**